### PR TITLE
take comment in account

### PR DESCRIPTION
**Pull request by NanoPish**

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@
     protocol: "{{ item.protocol | default('any') }}"
     route: "{{ item.route | default(omit) }}"
     log: "{{ item.log | default(false) }}"
+    comment: "{{ item.comment | default(omit) }}"
```
Comment (tersmitten) [inline tasks/configure.yml:66]: Can you also update the `README`?

Comment (NanoPish) [reply tasks/configure.yml:66]: sure! forgot

Comment (NanoPish) [reply tasks/configure.yml:66]: done

File: README.md [hunk 2]
```diff
@@ -62,6 +63,7 @@ None
       - rule: allow
         interface: eth1
         to_port: ''
+	comment: 'allow all kind of traffic on interface eth1'
```
Comment (tersmitten) [inline README.md:66]: This looks a bit odd

**Comment by NanoPish** [comment]
changed whitespace and sentence

**Pull request merged**